Add reputation based DAOs to DAO page (#5679)
ethereum/ethereum-org-website: src/content/dao/index.md

- A charity – you can accept membership and donations from anyone in the world and the group can decide how they want to spend donations.
- A freelancer network – you could create a network of contractors who pool their funds for office spaces and software subscriptions.
- Ventures and grants – you could create a venture fund that pools investment capital and votes on ventures to back. Repaid money could later be redistributed amongst DAO-members.

> - Protocol/product development & governance - you can form a global decentralized community collaborating trustlessly in the development and governance of protocols and applications.

Reviewer: Let's explain this below, the complexity jump here is too high from previous examples given our target audience of this page.

```suggestion
```

[MolochDAO](http://molochdao.com/) – MolochDAO is focused on funding Ethereum projects. They require a proposal for membership so the group can assess whether you have the necessary expertise and capital to make informed judgments about potential grantees. You can't just buy access to the DAO on the open market.

### Reputation-based membership {#reputation-based-membership}

> Reputation represents ‘proof-of-work’ and grants voting power, but unlike shares - no ownership, and members do not have a claim on the treasury. Unlike liquid tokens, reputation cannot be bought, sold, transferred or delegated - it is only earned, but it can also be slashed. Reputation-based DAOs do not require reputation or tokens for someone to submit a proposal for on-chain voting, it is permissionless. Prospective members can freely submit proposals to join the DAO, and in exchange for their contributions request to receive reputation and tokens as a reward.

Reviewer: ```suggestion
Reputation represents proof of participation and grants voting power in the DAO. Unlike token or share-based membership, reputation-based DAOs don't transfer ownership to contributors. Reputation cannot be bought, transferred or delegated; DAO members must earn reputation through participation. On-chain voting is permissionless and prospective members can freely submit proposals to join the DAO and request to receive reputation and tokens as a reward in exchange for their contributions.
```

> Due to their unique nature, wich doesn't share the constraints of token governance, reputation-based systems allow DAOs to expand to multiple networks and conduct cross-chain governance, without relying on bridges. That paired with holographic consensus also allows for DAOs to scale without compromising on resiliency.

Reviewer: ```suggestion
```

> _Typical for decentralized development and governance of protocols and dApps, but also well suited to a diverse set of organizations like charities, worker collectives, investment clubs, etc._

Reviewer: ```suggestion
_Typically used for decentralized development and governance of protocols and dApps, but also well suited to a diverse set of organizations like charities, worker collectives, investment clubs, etc._
```

#### A famous example {#reputation-example}

> [DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective that has been building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and mananage funds trustlessly. It's not possible for anyone to buy they way into influencing its future.

Reviewer: ```suggestion
[DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and manage funds, meaning no one can buy their way into influencing its future.
```

- [What is a DAO and what is it for?](https://daohaus.substack.com/p/-what-is-a-dao-and-what-is-it-for) – [DAOhaus](https://daohaus.club/)
- [How to Start a DAO-Powered Digital Community](https://daohaus.substack.com/p/four-and-a-half-steps-to-start-a) – [DAOhaus](https://daohaus.club/)
- [What is a DAO?](https://coinmarketcap.com/alexandria/article/what-is-a-dao) – [Coinmarketcap](https://coinmarketcap.com)
- [What is Holographic Consensus?](https://medium.com/daostack/holographic-consensus-part-1-116a73ba1e1c) - [DAOstack](https://daostack.io/)
- [Tokens, reputation, or maybe both?](https://forum.bprotocol.org/t/tokenized-governance/19/26) [DXdao](https://dxdocs.eth.link/)

### Videos {#videos}

- [What is a DAO in crypto?](https://youtu.be/KHm0uUPqmVE)

> - [DAOs ownining decentralized frontends](https://www.youtube.com/watch?v=2V6OHbYjCVg)

Reviewer: I don't think this adds much value to this page.

```suggestion
```
